src/EnergyPlus/SetPointManager.cc

```diff
@@ -197,37 +198,78 @@ constexpr std::array<std::string_view, static_cast<int>(SetPointManagerType::Num
+    DataLoopNode::ConnectionObjectType::SetpointManagerScheduled,
+    DataLoopNode::ConnectionObjectType::SetpointManagerScheduledDualSetpoint,
+    DataLoopNode::ConnectionObjectType::SetpointManagerOutdoorAirReset,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneReheat,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneHeating,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneCooling,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneHumidityMinimum,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneHumidityMaximum,
+    DataLoopNode::ConnectionObjectType::SetpointManagerMixedAir,
+    DataLoopNode::ConnectionObjectType::SetpointManagerOutdoorAirPretreat,
+    DataLoopNode::ConnectionObjectType::SetpointManagerWarmest,
+    DataLoopNode::ConnectionObjectType::SetpointManagerColdest,
+    DataLoopNode::ConnectionObjectType::SetpointManagerWarmestTemperatureFlow,
+    DataLoopNode::ConnectionObjectType::Invalid, // SPMType::ReturnAirBypass
+    DataLoopNode::ConnectionObjectType::SetpointManagerMultiZoneCoolingAverage,
+    DataLoopNode::ConnectionObjectType::SetpointManagerMultiZoneHeatingAverage,
+    DataLoopNode::ConnectionObjectType::SetpointManagerMultiZoneMinimumHumidityAverage,
+    DataLoopNode::ConnectionObjectType::SetpointManagerMultiZoneMaximumHumidityAverage,
+    DataLoopNode::ConnectionObjectType::SetpointManagerMultiZoneHumidityMinimum,
+    DataLoopNode::ConnectionObjectType::SetpointManagerMultiZoneHumidityMaximum, 
+    DataLoopNode::ConnectionObjectType::SetpointManagerFollowOutdoorAirTemperature,
+    DataLoopNode::ConnectionObjectType::SetpointManagerFollowSystemNodeTemperature,
+    DataLoopNode::ConnectionObjectType::SetpointManagerFollowGroundTemperature,
+    DataLoopNode::ConnectionObjectType::SetpointManagerCondenserEnteringReset,
+    DataLoopNode::ConnectionObjectType::SetpointManagerCondenserEnteringResetIdeal,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneOneStageCooling,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSingleZoneOneStageHeating,
+    DataLoopNode::ConnectionObjectType::SetpointManagerReturnTemperatureChilledWater,
+    DataLoopNode::ConnectionObjectType::SetpointManagerReturnTemperatureHotWater,
+    DataLoopNode::ConnectionObjectType::Invalid, // SPMType::TESScheduled 
+    DataLoopNode::ConnectionObjectType::SetpointManagerSystemNodeResetTemperature,
+    DataLoopNode::ConnectionObjectType::SetpointManagerSystemNodeResetHumidity
+};
+
+constexpr std::array<std::string_view, (int)SupplyFlowTempStrategy::Num> supplyFlowTempStrategyNamesUC = {
+    "MAXIMUMTEMPERATURE", "MINIMUMTEMPERATURE"};
+        
+// Why?
+constexpr std::array<std::string_view, (int)AirTempType::Num> oaTempTypeNamesUC = {
+    "OUTDOORAIRWETBULB", "OUTDOORAIRDRYBULB"};
+
+// No really, why?
+constexpr std::array<std::string_view, (int)AirTempType::Num> nodeTempTypeNamesUC = {
+    "NODEWETBULB", "NODEDRYBULB"};
+
```

Comment:
Why are these called `OutdoorAirXBulb` and `NodeXBulb` when they both just mean `WetBulb`? Gratuitous differences result in gratuitous code.